pkg/importer/dataStream.go

```diff
@@ -81,6 +81,7 @@ func (d *dataStream) http() (io.ReadCloser, error) {
 	}
 	req, err := http.NewRequest("GET", d.url.String(), nil)
 	if len(d.accessKeyId) > 0 && len(d.secretKey) > 0 {
+		req.SetBasicAuth(d.accessKeyId, d.secretKey)
```

Comment:
@copejon should comment here. I do not remember if this is what he intended to add, or whether he had another plan.

Reply:
It was really intended to handle CDN downloads, which have since been proven to require more manual effort.  It might still be useful for sites that prompt for user name and password for a download, but that has not been tested.

It is safe to remove this code, but if you, please also remove lines 77-80

Reply:
I got a question for that CDN stuff here: https://trello.com/c/Zd1E3IBe/100-13-epic-image-registry-automate-import-of-images-from-red-hat-cdn-8, not sure if that currently works or not ?  so my positive test is setting up a simple http auth server using python, which is case here:  https://polarion.engineering.redhat.com/polarion/#/project/CNV/workitems?query=shiywang, please check CNV-238

Reply:
Thank you for your initiative @shiywang ! I recall Jon mentioning some issues regarding using RH CDN (maybe `yum` is part of the process?). Anyway, he can comment on the above trello card.

Reply:
Unfortunately, we can't send an http request to a CDN URL with authentication attached.  The CDN generates download URLs dynamically, and they only live for 30 minutes.  This means that to use CDI with the CDN, you have to use the CDN web gui to navigate to the image you want, copy the download URL and paste that into the PVC annotation.  Authentication is not required to start the download.